draft-ietf-moq-transport.md

```diff
@@ -832,21 +857,21 @@ guarantee described in {{track-scope}}.
 Discovery of MoQT servers is always done out-of-band. Namespace discovery can be
 done in the context of an established MoQT session.
 
-Given sufficient out of band information, it is valid for a subscriber
-to send a SUBSCRIBE or FETCH message to a publisher (including a relay) without
-any previous MoQT messages besides SETUP. However, SUBSCRIBE_ANNOUNCES and
+Given sufficient out of band information, it is valid for a subscriber to send a
+SUBSCRIBE or FETCH message to a publisher (including a relay) without any
+previous MoQT messages besides SETUP. However, SUBSCRIBE_ANNOUNCES, PUBLISH and
 ANNOUNCE messages provide an in-band means of discovery of publishers for a
 namespace.
 
 The syntax of these messages is described in {{message}}.
 
 
-## Subscribing to Announcements
+## Subscribing to Namespaces
 
 If the subscriber is aware of a namespace of interest, it can send
 SUBSCRIBE_ANNOUNCES to publishers/relays it has established a session with. The
```

Comment:
Should we rename to Subscribe_Namespaces ?

Reply:
Ian asked me not to because it bloats the diff and makes it hard to read.  We will follow up, either by renaming, or moving to SUBSCRIBE.